packages/daemon/package.json

```diff
@@ -20,7 +21,7 @@
     "test_images_wait_for_db": "yarn dlx ts-node ./__tests__/integration/scripts/wait-for-db-up.ts",
     "test_images_wait_for_ws": "yarn dlx ts-node ./__tests__/integration/scripts/wait-for-ws-up.ts",
     "test_images_setup_database": "yarn dlx ts-node ./__tests__/integration/scripts/setup-database.ts",
-    "test": "jest --coverage",
+    "test": "jest --coverage --verbose",
```

Comment:
question(non-blocking): Should we keep the `--verbose` flag here?

Reply:
No! Removed it in 3e1a48f162c8755e6042000f17493567e414a534

Thanks